frontend/nuxt.config.js

```diff
@@ -38,8 +39,12 @@ export default {
             { property: 'og:type', content: 'website' },
             { property: 'og:description', content: SITE_DESCRIPTION },
             { property: 'og:image', content: '/preview.jpeg' },
-            { name: 'twitter:card', content: 'summary_large_image' },
             { name: 'theme-color', content: '#1aab9b' },
+            { name: 'twitter:card', content: 'app' },
+            { name: 'twitter:site', content: TWITTER_HANDEL },
+            { name: 'twitter:title', content: SITE_TITLE },
+            { name: 'twitter:description', content: SITE_DESCRIPTION },
+            { name: 'twitter:image', content: '/preview.jpeg' },
```

Comment:
Meta-tags usually do not support relative paths (same for the other `og:image` tag)
https://stackoverflow.com/a/45941439

Reply:
This is not a relative path in this case. We have the image `preview.jpeg` in our static folder. When the frontend is started all the images and files in the static folder are served. Meaning that the image is available in this way (it also works on discord and slack). 

I assume that Twitter was just missing the `twitter:image` tag. Does this explanation make sense?

Reply:
I mean that twitter expects `twitter:image` or `og:image` tag to include domain name, eg `https://domain.com/path/to/the/image.jpg` at least that's how it was some time ago: https://stackoverflow.com/a/9858694

Reply:
Hmm, okay that makes sense. 

Can we use the environment variable `PRODUCTION_DOMAIN` as the base URL? Or is there an even easier way I can retrieve the current domain? I am not sure how this works due to our Kubernetes Cluster (if the WebApp runs in containers it might not know its own domain right?)

Reply:
`PRODUCTION_DOMAIN` is a domain without schema and is used for different purpose as stated in the readme. I guess you would need to add env variable something like `FRONTEND_ORIGIN` for this. Please don't forget to modify:
- dockerfile
- drone
- nuxt.config
- readme
- secrets (or provide commands if you don't have enough rights)